File: app/src/main/java/com/myjb/dev/recyclerView/BookItemView.java
package com.myjb.dev.recyclerView;

import android.content.Context;
import android.view.LayoutInflater;

import com.myjb.dev.mygaragesale.databinding.ItemRecyclerviewBookBinding;

public class BookItemView extends ItemView {

    ItemRecyclerviewBookBinding binding;

    public BookItemView(Context context) {
        super(context);
        binding = ItemRecyclerviewBookBinding.inflate(LayoutInflater.from(context));
        addView(binding.getRoot());
    }

    @Override
    public void bind(Object item) {
        if (item instanceof String) {
            binding.book.setText((String) item);
        }
    }
}

File: app/src/main/java/com/myjb/dev/view/recyclerView/BookItemView.kt
package com.myjb.dev.view.recyclerView

import androidx.recyclerview.widget.RecyclerView
import com.myjb.dev.mygaragesale.databinding.ItemRecyclerviewBookBinding

class BookItemView(private val view: ItemRecyclerviewBookBinding) :
    RecyclerView.ViewHolder(view.root) {

    fun bind(item: String) {
        view.book.text = item
    }
}